src/main/views/live-case.njk

```diff
@@ -0,0 +1,79 @@
+{% from "govuk/components/table/macro.njk" import govukTable as 'govukTable' %}

```

Comment:
this file would be renamed to `live-case-court-search` or `live-case-alphabet-search`